daml-script/daml/Daml/Script.daml

```diff
@@ -769,8 +769,8 @@ submitTreeMulti actAs readAs cmds =
 -- | HIDE Identifier for a user in the user management service
 newtype UserId = UserId Text deriving (Eq, Ord, Show)
```

Comment:
Instead of deriving the show instance here, maybe we should show the underlying string directly like we do for parties?

Reply:
with the single quotes?
LF.daml>
```
instance Show Party where show party = "'" <> partyToText party <> "'"
````

Reply:
yeah I’d probably go for the same quoting so it’s distinguishable from strings

Reply:
:+1: